packages/lnsp-mediator/importer/migrations/20250127000000-add-duplicate-prevention-and-error-handling.js

```diff
@@ -0,0 +1,84 @@
+module.exports = {
+  async up(db, client) {
+    console.log("Adding duplicate prevention and error handling features...");
+
+    // 1. Add indexes to laborders collection for efficient duplicate detection
+    console.log("Creating indexes for duplicate detection...");
+    await db.collection('laborders').createIndex(
+      { labOrderId: 1, patientId: 1, facilityId: 1 },
```

Comment:
Technically, SCC requires that `labOrderId` is unique.

Reply:
I think this is more from the isanteplus perspective, where this set of data in the hl7 message indicates it was sent multiple times from an isanteplus instance.